layout/_partial/_head-sections/script.ejs

```diff
@@ -24,8 +24,6 @@ var _busuanzi = theme.busuanzi_src || '/js/thirdParty/busuanzi.min.js';
 <% if(theme.gitment.enable && is_post()){ %>
 <%- js("/js/thirdParty/gitment.browser.min.js") %>
 <script>
-  var labels = "<%= theme.gitment.labels %>";

```

Comment:
If you remove the `labels`, it will use `gitment` default labels, which is not configurable in the theme. What problems you have met if you have **not** deleted the variables named lables?

Reply:
If I haven't delete it, I cann't initialize the first comment. I don't know whether the author of gitment  modify somthing. Once I delete the lables thing, it will pass.

Reply:
Would you mind try it?

Reply:
There are no problems when using custom labels.It seems that this is not a bug.Maybe there is something wrong with your config file.I think you could make a issue but not PR.